### PR TITLE
Add experimental support for db2.

### PR DESCRIPTION
**Pull request by kklein**

### CODE REVIEW

File: tests/integration/conftest.py [hunk 1]
```diff
@@ -111,7 +113,7 @@ def mix_table1(engine, metadata):
     table_name = "mix_table1"
     columns = [
         sa.Column("col_int", sa.Integer()),
-        sa.Column("col_varchar", sa.String()),
+        sa.Column("col_varchar", sa.String(40)),
```
Comment (ivergara) [inline tests/integration/conftest.py:116]: I wouldn't have expected this to break `impala`'s integration tests.

Comment (kklein) [reply tests/integration/conftest.py:116]: Haha I wasn't exactly hoping for this either. I'll try to figure something out... Worst case we need a case distinction.

**Comment by kklein** [comment]
FYI @FlorianOttQC I hope to have experimental support for db2 released soon.

**Comment by kklein** [comment]
> I'm kind of surprised that the tests didn't break with sqlalchemy 2.0.

I would suppose that we have some dependencies expecting sqlalchemy < 2.0

**Pull request merged**